pkg/cache/clusterqueue.go

```diff
@@ -250,6 +250,11 @@ func (c *clusterQueue) updateQueueStatus() {
 	if status != c.Status {
 		c.Status = status
 		metrics.ReportClusterQueueStatus(c.Name, c.Status)
+		if lqMetrics {
+			for _, lq := range c.localQueues {
```

Comment:
This iteration might be adding unnecessary performance cost. What is the scenario that it needs calling here? Maybe we could move the call per LQ, when we update the specific LQ. PTAL.

Reply:
The lq status is equal to the cq status. So when the cq status updates, all the cqs associated lqs should have their statuses updated as well